### PR TITLE
Can configure indexing policy for CosmosDB SQL Container

### PR DESCRIPTION
**Pull request by guillaumestevens**
Add indexing_policy block to configure Indexing Policy (especially included and excluded path) for a SQL Container in a CosmosDB Database.

### CODE REVIEW

File: azurerm/internal/services/cosmos/resource_arm_cosmosdb_sql_container.go [hunk 1]
```diff
@@ -75,6 +75,81 @@ func resourceArmCosmosDbSQLContainer() *schema.Resource {
 				ValidateFunc: validate.CosmosThroughput,
 			},
 
+			"indexing_policy": {
+				Type:       schema.TypeList,
+				Optional:   true,
+				Computed:   true,
```
Comment (jackofallops) [inline azurerm/internal/services/cosmos/resource_arm_cosmosdb_sql_container.go]: I think this should have a `MaxItems: 1,`

**Comment by guillaumestevens** [comment]
Hi, 

i'm working on integrate your comment. 

Guillaume

File: azurerm/internal/services/cosmos/resource_arm_cosmosdb_sql_container.go [hunk 2]
```diff
@@ -75,6 +75,81 @@ func resourceArmCosmosDbSQLContainer() *schema.Resource {
...
+						"automatic": {
+							Type:     schema.TypeBool,
+							Optional: true,
+							Computed: true,
+						},
+						"included_paths": {
+							Type:       schema.TypeList,
+							Optional:   true,
+							Computed:   true,
+							ConfigMode: schema.SchemaConfigModeAttr,
```
[the remark this answers is not in the document]
Comment (katbyte) [reply azurerm/internal/services/cosmos/resource_arm_cosmosdb_sql_container.go:101]: For optional computed blocks that we need to be able to set to empty this is unfortunately needed due to how tf.12 works. which would also negate the name change for this property

**Comment by katbyte** [comment]
hey @guillaumestevens, thanks for this PR but as the review hasn't been addressed for 4 months and there are merge conflicts dueto the SDK upgrade i am going to close this PR. Please do reopen it if you have the time to finish it off.

**Pull request closed without merging**